framework/db/ActiveQuery.php

```diff
@@ -721,4 +724,35 @@ public function viaTable($tableName, $link, callable $callable = null)
 
         return $this;
     }
+
+    /**
+     * Define an alias for the table defined in [[modelClass]].
+     *
+     * This method will adjust [[from]] so that an already defined alias will be overwritten.
+     * If none was defined, [[from]] will be populated with the given alias.
+     *
+     * @param string $alias the table alias.
+     * @return $this the query object itself
+     * @since 2.0.7
+     */
+    public function alias($alias)
```

Comment:
Not sure, `alias` in context of a query is more clear than `as`, it defines an alias for the main table in the query and not uses the query "as" something.